helm/nginx-ingress-controller-app/Chart.yaml

```diff
@@ -6,3 +6,5 @@ icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/5
 name: nginx-ingress-controller-app
 namespace: kube-system
 version: [[ .Version ]]
+sources:
+  - https://raw.githubusercontent.com/giantswarm/nginx-ingress-controller-app/v1.6.7/README.md
```

Comment:
```suggestion
  - https://raw.githubusercontent.com/giantswarm/nginx-ingress-controller-app/[[ .Version ]]/README.md

```
@oponder does this make sense and work for you? added a new line @ the EOF but more importantly made this link dynamic, automatically updated by the release process.

Reply:
Yes absolutely! Thanks stevo!